lib/cryptoexchange/exchanges/unidax/services/market.rb

```diff
@@ -0,0 +1,38 @@
+module Cryptoexchange::Exchanges
+  module Unidax
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output['data'], market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Unidax::Market::API_URL}/get_ticker?symbol=#{market_pair.base.downcase}#{market_pair.target.downcase}"
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Unidax::Market::NAME
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.high      = NumericHelper.to_d(output['high'])
+          ticker.low       = NumericHelper.to_d(output['low'])
+          ticker.bid       = NumericHelper.to_d(output['buy'])
+          ticker.ask       = NumericHelper.to_d(output['sell'])
+          ticker.volume    = NumericHelper.to_d(output['vol'])
+          ticker.timestamp = NumericHelper.to_d(output['time']) / 1000
```

Comment:
lets set this to `nil`, as we faced tonnes of issue relying on the returned timestamp

Reply:
done